### ISSUE TITLE
NCD group migration to new TDWG site 

### ISSUE THREAD

**Issue by pmergen** [tdwg/infrastructure#37]
1st suggestion discussed with Wouter Addink on NCD group : This group is not active any more and could be archived as former group. The task discuss here is now taken on on a larger scale with GRBIO (http://grbio.org/) . the suggestion is to contact David Schindel and ask if for NCD a link can be put on the TDWG website from the NCD page to GRBIO and work on revive the Natural Collection Description Metadata of TDWG together with the GRBIO. 
Opinion is needed here if this is the way forward ?

Pat

**Comment by CynthiaParr-USDA** [comment]
Let's list the group as not active, and I agree it would be helpful to put link to GRBio.  It may become active in the future. For this reason, we are suggesting that all inactive groups appear on the activities table. I guess inactive groups should not appear in the side bar?

If you want the executive to work on reviving it, you could make a ticket in the executive group, if you want others to see the suggestion, you could open a ticket in your new repository (which is open) you could try it there.

**Comment by peterdesmet** [comment]
We need a guideline on what to do with inactive groups. Keeping them around in the main activities list (and especially sidebar) will clutter things, but we should archive them somewhere. Most conveniently would be to archive them in typo3, as that is where they are created, so it's just a matter of moving a page.

**Comment by CynthiaParr-USDA** [comment]
Definitely they should be taken off the sidebar, but I would propose that we make an "inactive" table on this page so that people can find them without having to Google http://www.tdwg.org/activities/

**Comment by peterdesmet** [comment]
Having an inactive table on http://www.tdwg.org/activities/ and not listing them in sidebar seems good to me.

**Comment by pmergen** [comment]
yes, we could have like in the standards a place for archived activities groups or call them former groups. would avoid the term "inactive" in the wording on the site, does not sound so good.  especially this one, if we put the link to GRBIO in discussion if it can be revived in collaboration with them. :http://grbio.org/ to discuss within the exec and then contact them ...

**Comment by CynthiaParr-USDA** [comment]
Let's move ahead with this:
- [ ] Create a table for inactive groups, with NCD the first member (there may be older ones)
- [ ] Remove NCD from the left side bar
- [ ] Add mention of GRBio to the NCD page
- [ ] Remove Wouter Addink's name as convener from NCD page (it could say "Pending")

**Comment by CynthiaParr-USDA** [comment]
Stan, can you tackle this and close the ticket as part of the maintenance of the new website? Thanks.

**Comment by MattBlissett** [comment]
Old page is available at `http://old.tdwg.org/activities/`.

(And assigning the real Stan.)
